.github/workflows/release-build-publish.yml

```diff
@@ -20,17 +20,15 @@ jobs:
       IS_TEST: ${{ github.event.inputs.publishing_environment == 'testnet' }}
       OS_GRPC_URL: https://test.figure.com:80
       PROVENANCE_GRPC_URL: grpcs://34.148.39.82:9090
-#      ENCRYPTION_PRIVATE_KEY: IS_TEST && ${{ secrets.TEST_PRIVATE_KEY }} || ${{ secrets.PROD_PRIVATE_KEY }}
-#      CHAIN_ID: IS_TEST && "pio-testnet-1 || "pio-mainnet-1"
-      ENCRYPTION_PRIVATE_KEY: ${{ secrets.TEST_PRIVATE_KEY }}
-      SIGNING_PRIVATE_KEY: ${{ secrets.TEST_PRIVATE_KEY }}
-      PROD_ENCRYPTION_PRIVATE_KEY: ${{ secrets.PROD_PRIVATE_KEY }}
-      PROD_SIGNING_PRIVATE_KEY: ${{ secrets.PROD_PRIVATE_KEY }}
-      TEST_CHAIN_ID: pio-testnet-1
-      PROD_CHAIN_ID: pio-mainnet-1
+      ENCRYPTION_PRIVATE_KEY: IS_TEST && ${{ secrets.TEST_PRIVATE_KEY }} || ${{ secrets.PROD_PRIVATE_KEY }}
```

Comment:
Was seeing some issues with github evaluating this correctly. Will revisit when bootstrapping from here actually works :sadpanda:

Reply:
When I made the suggestion in #2, I hadn't actually checked what the proper syntax was 😅  I can take a stab at it in another PR

Reply:
i feel betrayed. But ya it's all good for now - will have to publish things locally until pierce can get a kong gateway open for us to use.